### PR TITLE
Adding decision to annex mongolia 39

### PR DESCRIPTION
**Pull request by GrenadierSchubes**
Mongola annexation event 39

### CODE REVIEW

**Comment by Narizna** [comment]
I have taken the stance a number of times in the past that mongolia should just be removed as a player.

it is arguably crutch for Russia if they don't have one
it is too powerful as an annexed nation for Russia

File: common/decisions/SOV.txt [hunk 1]
```diff
@@ -5541,6 +5541,46 @@ SOV_great_patriotic_war = {
...
+			tag = SOV			
+		}
+		
+		visible = {
+			tag = SOV
+			MON = { is_ai = yes }
+			country_exists = MON
+		}
+		available = {
+			date > 1939.1.1
```
Comment (Narizna) [inline common/decisions/SOV.txt:5558]: I would make 1941 and then would accept proposal

**Review by Narizna — CHANGES_REQUESTED** [review]

File: common/decisions/SOV.txt (hunk 1 above)
Comment (iDealNimzi) [reply common/decisions/SOV.txt:5558]: that seems fair to try.

File: common/decisions/SOV.txt [hunk 2]
```diff
@@ -5541,6 +5541,46 @@ SOV_great_patriotic_war = {
...
+
+		complete_effect = {
+			MON = {
+				transfer_navy = {
+					target = SOV
+				}
+			}
+			annex_country = {
+              target = MON
+              transfer_troops = yes
```
Comment (iDealNimzi) [inline common/decisions/SOV.txt:5573]: I like the transfer_troops as well....

Is it possible to make this event available if 1 of the big cities fall?

File: common/decisions/SOV.txt (hunk 1 above)
Comment (Mookubooku) [reply common/decisions/SOV.txt:5558]: I like the 1941 annex personally. We buffed Mongolia way more than it should be so a 1939 annex seems too strong unless we bring Mongolia's factory counts back down significantly.

**Review by GrenadierSchubes — COMMENTED** [review]
changed start date

**Review by Ruling-0 — APPROVED** [review]
Assuming this is tested

**Comment by iDealNimzi** [comment]
Did this get tested?

**Comment by GrenadierSchubes** [comment]
Testing works and localization is live

**Review by Mookubooku — APPROVED** [review]
1941 decision for SOV to annex MON looks good.

File: common/decisions/SOV.txt (hunk 1 above)
Comment (GrenadierSchubes) [reply common/decisions/SOV.txt]: Changed to 41 awaiting approval.

**Review by lolmemes369 — CHANGES_REQUESTED** [review]
I think the issue with annex builds and annexing Ai in general is that you start out with 100% compliance or can get it pretty easily. not exactly sure on the math. But the high compliance could make annex builds just better. This applies not just to the soviets but other nations. If 100% compliance makes up for annexing mongolia in 1941 and not having a player on it then the changes should be fine.

**Comment by GrenadierSchubes** [comment]
Will come to discussion on annex build reduce compliance once axis are lowered as well.  For now they will all be the same.

**Pull request merged**